apps/performance/ngfor-biglist/src/app/person-list.component.ts

```diff
@@ -27,4 +31,8 @@ import { Person } from './person.model';
 })
 export class PersonListComponent {
   @Input() persons: Person[] = [];
+
+  trackPerson(index: number, person: Person) {
+    return person ? person.email : undefined;
```

Comment:
question: why did you do this ternary statement? Can person be undefined?

Reply:
Copy & Pasted from another project.  I forgot to add the index parameter so I just grabbed a similar trackBy I had recently used.

Reply:
You can also use my trackBy directive that simplify the work. `libs/shared/directives/src/lib/track-by.directive.ts`